crates/oxc_ast/src/ast/js.rs

```diff
@@ -269,6 +275,8 @@ pub struct BindingIdentifier {
     #[cfg_attr(feature = "serde", serde(flatten))]
     pub span: Span,
     pub name: Atom,
+    #[cfg_attr(feature = "serde", serde(skip))]
+    pub symbol_id: Option<SymbolId>,
```

Comment:
@Boshen Do you think `Option<SymbolId>` is bad for DX? Or you would prefer  to have `Symbol::invalid()` as the default value for non-resolved `Identifier` nodes.

Reply:
Let's not change APIs in the same PR. I comment here should do the job.

Reply:
I don't follow. Did you mean using `pub symbol_id: SymbolId`?